src/utils.ts

```diff
@@ -29,3 +32,22 @@ export function epochToBatchId(epoch: BigInt): BigInt {
 export function getBatchId(event: EthereumEvent): BigInt {
   return epochToBatchId(event.block.timestamp)
 }
+
+export function getOwlDecimals(): BigInt {
+  return OWL_DECIMALS
+}
```

Comment:
Apparently yes, it didn't allow me to export the `const`, had to do the function. didn't like it either. If you know how to do it let me know...

Reply:
Hmm odd. Anything to do with the fact that the variables are declared with `let` instead of `const`?:

```
let OWL_DECIMALS = BigInt.fromI32(18)
```

BTW, why `let` in the first place?

Reply:
cause assembly script don't let us use that. Look how they do it https://thegraph.com/docs/define-a-subgraph#writing-mappings